src/DependencyInjection/TacticianDomainEventExtension.php

```diff
@@ -71,4 +77,17 @@ private function addSubscribers(ContainerBuilder $container)
             ]);
         }
     }
+
+    private function registerEventCollector(ContainerBuilder $container, $useIdentityMap)
+    {
+        $class = 'BornFree\TacticianDoctrineDomainEvent\EventListener\CollectsEventsFromEntities';
```

Comment:
is it possible to use `CollectsEventsFromEntities::class` instead? and on line 85

Reply:
Sure, but then I need to change PHP requirement from 5.4 to at least 5.5. Are you fine with it?

Reply:
Omg, it's 5.4 o_O. http://php.net/supported-versions.php According to this table, I would suggest 7.0+, but let's have 5.6+ since it's still supported for security fixes. What do you think?